scenarios/27_Orannon.cfg

```diff
@@ -1642,9 +1642,9 @@ Forces of Darkness will consume your mind."}
 
     # A good portion of this scenario's code is actually in this FINAL_INTERROGATIONS macro;
     # it is located in macros/ano-20macros.cfg:
```

Comment:
oh right, I meant to mention: if moving the macro, we should also move the comment associated with it, too